.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1457,7 +1457,7 @@ dynamodb_Scenario_GettingStartedMovies:
     PHP:
       versions:
         - sdk_version: 3
-          github: php/example_code/dynamodb/dynamodb_basics
+          github: php/example_code/dynamodb

```

Comment:
Need to make this same change to all of the PHP github links in this file.

Reply:
Gotcha, fixed across the board now